static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -3,12 +3,26 @@ import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 /**
  * Generate breadcrumb descriptions based on type
  */
-export function getDescription(crumb: Crumb) {
+export function getDescription(crumb: Crumb, startTimestampMs?: number) {
+  if (crumb.data && 'action' in crumb.data) {
+    switch (crumb.data.action) {
+      case 'largest-contentful-paint':
+        if (crumb.timestamp && startTimestampMs) {
+          return `${new Date(crumb.timestamp).getTime() - startTimestampMs}ms`;
+        }
+        break;
```

Comment:
Unneeded

Reply:
Did you add this into the SDK?

Reply:
Was referring to the break but we should leave it